### PR TITLE
Usmerge

### PR DESCRIPTION
**Pull request by isantop**
Pull in latest upstream changes.

Updates the remove dialog to better match GNOME HIG and to not run on for as long. 

Adds more descriptive text to policykit authentication dialog when Repoman is launched.

### CODE REVIEW

File: debian/repoman.debhelper.log [hunk 1]
```diff
@@ -0,0 +1,18 @@
+dh_update_autotools_config
```
Comment (jackpot51) [inline debian/repoman.debhelper.log]: This file should not be committed

File: debian/repoman/DEBIAN/control [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Package: repoman
```
Comment (jackpot51) [inline debian/repoman/DEBIAN/control]: This folder should not be committed

File: debian/repoman/usr/bin/repoman [hunk 3]
```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/python3
```
Comment (jackpot51) [inline debian/repoman/usr/bin/repoman]: This folder should not be committed

**Review by jackpot51 — DISMISSED** [review]
You need to run `dh clean` before committing. You have leftover files that should not be in the repository

**Review by jackpot51 — APPROVED** [review]
Looks good now

**Review by brs17 — APPROVED** [review]
lgtm

**Comment by jackpot51** [comment]
@isantop if you squash and merge, it will keep the debian files out of history.

**Comment by isantop** [comment]
Done

**Pull request merged**